Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -8,11 +8,16 @@ private let BorderColor = UIColor(rgb: 0xeeeeee)
 private let LabelFont = UIFont(name: "FiraSans-Regular", size: 11)
 private let LabelColor = UIAccessibilityDarkerSystemColorsEnabled() ? UIColor.blackColor() : UIColor.darkGrayColor()
 private let CellInsets = UIEdgeInsetsMake(8, 8, 8, 8)
-private let TextMargin = 5
+
+private let PlaceholderImage = UIImage(named: "defaultFavicon")
+
+struct ThumbnailCellUX {
+    static let ImageAspectRatio: Float = 1.5
```

Comment:
I'm confused about the difference between the aspect ratio and the one used in TopSitesPanel. Are they different? Just by the label height on the bottom of the thumbnail?